### ISSUE TITLE
Add an abbreviated Top-10 focused guide

### ISSUE THREAD

**Issue by J3rry367** [OWASP/OWASP-Testing-Guide-v5#171]
As a consultant, I run into a lot of smaller groups who aren't looking to pay for the number of hours it takes to work through the full ASVS. This is especially the case when the client has multiple web apps they need to have reviewed, we end up having to submit bids for abbreviated reviews where we focus solely on the Top-10. 

It would be great if there was a quick and dirty testing guide that could be offered as a bare minimum assessment or even as a way to show that they really should have a full review performed. 

I'd looked into the cheetsheet series, but the Top-10 cheetsheet has been recommended for deletion. While I understand the thinking behind the Top-10 being primarily for education purposes rather than a testing guide, in my experience there's a definite need for something like this.

We created a quick work program based off of the 2017 Top-10 that I'd love some feedback on and/or could be used as a jumping-off point for this.
[OWASP-TOP10.xlsx](https://github.com/OWASP/OWASP-Testing-Guide-v5/files/3600989/OWASP-TOP10.xlsx)

**Comment by kingthorin** [comment]
I think we could provide a mapping of the Testing Guide to the Top 10. 

However, doing an assessment based solely on Top 10 items still leaves a huge amount of attack surface or abuse cases (some of which may be more important over the top 10's 2 or 3 year lifespan, or due to new research). We can probably add some commentary to that effect in the intro sections....

**Comment by victoriadrake** [comment]
@J3rry367 How do you see this integrating into the WSTG? It's not focused on specific tests, so if it's a guide, what is the criteria for having followed it?

**Comment by ThunderSon** [comment]
@victoriadrake I don't think it's creating new test cases. It's creating a sheet that covers the Top 10.
It's a simple list of simply the top 10 vulnerabilities, which we can create from the map we have between issues and test cases.

**Comment by J3rry367** [comment]
@victoriadrake yeah, the lack of specific tests is exactly what I'm running into. Like @ThunderSon said I don't think anything new needs to be added, rather something like "here's the top 10(ish) vulnerabilities to check for based on what is currently prevalent" or even "at the bare minimum all apps should have these controls." I guess, basically, I'm thinking something similar to how the MSTG has three levels of testing, a baseline which all apps need, then level 2, and reverse engineering for more sensitive apps. We work with sensitive apps, generally, and don't recommend the baseline, but if someone really needed something relatively quick and cheap we have that to offer.

**Comment by kingthorin** [comment]
> "at the bare minimum all apps should have these controls."

Is addressed by the Proactive Controls project: https://owasp.org/www-project-proactive-controls/ (See 'Quick Access' in the right nav sidebar)

**Comment by victoriadrake** [comment]
I could see this becoming a broad overview section, maybe as part of the [proposed](https://github.com/OWASP/wstg/pull/254/files) (#254) `Introduction_to_Web_Testing` section grouping. @J3rry367 are you willing to point-form a sort of Top 10 overview article so we can better envision it?

**Comment by J3rry367** [comment]
Thanks @kingthorin I didn't know that existed I will definitely check that out. 
Sure, @victoriadrake I can take a stab at it based on what I've been seeing requested.

**Comment by rahuldotbiz** [comment]
can I work on this?

**Comment by anuranjansb** [comment]
“Hi! I’m currently exploring OWASP projects for GSoC and found this discussion insightful.
I agree that a lightweight Top-10-based testing checklist could bridge the gap for smaller orgs.
I’d love to help organize or document a draft version of such a guide if that would be helpful.”

**Comment by suii2210** [comment]
Hi! I’d like to work on this issue.

Based on the discussion above, I will draft a simple, point-form
Top-10 overview that maps each Top-10 item to relevant WSTG
sections, along with a short intro explaining its purpose and limitations.

Once the draft is ready, I’ll submit a PR for review.

**Comment by kingthorin** [comment]
First off the issue hasn't been accepted yet. (Edit: maybe it's more accurate to say that the solution hasn't been agreed upon)

Second it is already assigned to someone.

Thirdly the PR you sent is full of halucinations and AI slop.

**Comment by suii2210** [comment]
ok , sure

**Comment by kingthorin** [comment]
I guess something like this could be added directly to the web deployment and live in that repo.

I don’t really see it being part of the guide project itself.

It should also use the 2025 release of the Top 10 (currently RC, so may as well wait for full release). Should properly link other docs (ex ASVS) and WSTG content (ex: no hallucinated testing scenarios… it’s okay if the Top 10 isn’t fully covered), and should be VERY clear about the negatives/downfall of testing only against the Top 10 “risks”.

**Comment by rbsec** [comment]
I kind of hate this on principle, because it always bugs me when I see clients asking for "OWASP Top 10 pentests", and just ignoring everything else. I can see how some kind of mapping/checklist might be useful, but as you say it needs big caveats about why this is a bad idea.

**Comment by kingthorin** [comment]
I'm good with closing it.

Someone could take it to the Top 10 project as an idea for them to adopt/address.
